app/modules/training/http_training.py

```diff
@@ -0,0 +1,55 @@
+import webapp2
+import logging
+import os
+import requests
+from google.appengine.ext.webapp import template
+from requests_toolbelt.adapters import appengine
+
+
+appengine.monkeypatch(validate_certificate=False)
```

Comment:
Why do you require this "monkeypatch"? What error text do you see when this is not present.

Reply:
this error occurs if its removed:
SSLError("Can't connect to HTTPS URL because the SSL module is not available.",))

Reply:
Understood @Najipian. Please add a comment above this referencing the documentation for this:
https://toolbelt.readthedocs.io/en/latest/adapters.html#appengineadapter

Also, please avoid having more than one blank line between code blocks.